### PR TITLE
OTdumnS8 assigning buses to routes

### PR DESCRIPTION
**Pull request by samuel776**
#### What does this PR do?
 - assign buses to routes
 - view list of buses and their respective buses
#### Description of Task to be completed?
- finding the routes in database by their id
- buses should be assigned to the routes
- user should be able to view list of buses to their respective routes
#### How should this be manually tested?
- clone it to the local repo
- run npm install
- test the end points in postman: `/api/assignments`
#### What are the relevant trello id?
[OTdumnS8](https://trello.com/c/OTdumnS8/13-assign-buses-to-routes)
#### Screenshots 
image

### CODE REVIEW

**Review by GuevaraMarius — APPROVED** [review]

**Review by mansurissa — APPROVED** [review]
Nice Job man

**Review by KundwaBruno — APPROVED** [review]

File: src/controllers/locales/en.json [hunk 1]
```diff
@@ -1,100 +1,115 @@
...
+	"there was an error while getting all assigned buses": "there was an error while getting all assigned buses",
+	"Bus is already assigned to this route": "Bus is already assigned to this route",
+	"There was an error while assigning bus": "There was an error while assigning bus",
+	"Bus is not yet assigned to any route": "Bus is not yet assigned to any route",
+	"unAssigned bus from route successfully": "unAssigned bus from route successfully",
+	"There was an error while unassigning bus": "There was an error while unassigning bus",
+	"Validation error: routeID is not allowed": "Validation error: routeID is not allowed",
+	"Validation error: distance must be a string": "Validation error: distance must be a string",
+	"Validation error: busStops is not allowed": "Validation error: busStops is not allowed"
+}
```
Comment (raymond42) [inline src/controllers/locales/en.json]: let's have an empty line at the end file

**Comment by mansurissa** [comment]
Nice job you should change some how in description of this PR

File: src/controllers/locales/en.json (hunk 1 above)
Comment (mansurissa) [reply src/controllers/locales/en.json]: Yeah sure let me work us that

**Review by raymond42 — APPROVED** [review]

**Pull request merged**